lib/tiny-lru.d.ts

```diff
@@ -1,14 +0,0 @@
-declare module "tiny-lru" {
-  export class Lru<T = any> {
-    constructor(max?: number, ttl?: number);
-
-    public has(key: string): boolean;
-    public get(key: string): T;
-    public set(key: string, value: T, bypass?: boolean): this;
-    public clear(): this;
-    public delete(key: string): this;
-    public evict(): this;
-    public keys(): string[];
-  }
-  export default Lru;
-}
```

Comment:
it seems this was just copied from root to lib, which looks un-necessary. The type file at root works perfectly too if `package.json` has `types` key.